site/src/pages/components/selectmenu.mdx

```diff
@@ -84,7 +84,9 @@ The above example results in the following style:
   alt="The rendering of a selectmenu control with the above HTML and CSS code"
 />
 
-`::part()` can be used to style the `button`, `selected-value`, and `listbox` parts of the control.
+`::behavior()` can be used to style the `button`, `selected-value`, and `listbox` parts of the control.
+
+Note that the `::part()` pseudo-element selector only matches elements in the user-agent shadow tree, and hence, fails to match any author-provided part (as in [slotting your own content](#slotting-your-own-content)).
```

Comment:
can you please wrap this with a `div` with a class of `.note`

Reply:
Also see my above comment on whether `::behavior` is defined somewhere as this should be covered in that definition; not in `selectmenu` itself.